services/ui/ws/platform_display_default.cc

```diff
@@ -162,8 +166,8 @@ void PlatformDisplayDefault::UpdateEventRootLocation(ui::LocatedEvent* event) {
 }
 
 void PlatformDisplayDefault::OnBoundsChanged(const gfx::Rect& new_bounds) {
-  // We only care if the window size has changed.
-  if (new_bounds.size() == metrics_.bounds_in_pixels.size())
+  // We only care if the window size has changed and |image_cursors_| exists.
+  if (new_bounds == metrics_.bounds_in_pixels && !image_cursors_)
```

Comment:
I am not 100% is null-check the cursor here is the best place.

maybe we should bail out earlier in case image_cursors_ is NULL?